charts/topolvm/values.yaml

```diff
@@ -175,10 +175,13 @@ lvmd:
   #      path: /run/topolvm
   #      type: DirectoryOrCreate
 
-  # lvmd.volumeMounts -- Specify volumeMounts.
-  volumeMounts: []
-  #  - name: lvmd-socket-dir
-  #    mountPath: /run/topolvm
+  # lvmd.additionalVolumes -- Specify additional volumes without conflicting with default useful for initContainers.
```

Comment:
I assume this sentence `useful for initContainers` should not be written as it may mislead users into thinking that it can only be used for certain use cases.

Reply:
There seems to be no fix or comment. Is this one an oversight in the correction?

Reply:
I think so.

@ryanfaircloth 
Could you please tell us your intentions?

Reply:
I simple misunderstood the feedback in the context of another feedback item I did resolve. I have added additional comments to clarify its intention is for initcontainers but it is not limited to that use case. At the moment that is the only use case I can imagine.